features/ssh_push.feature

```diff
@@ -1,11 +1,16 @@
-# Copyright (c) 2015 SUSE LLC
+# Copyright (c) 2016 SUSE LLC
 # Licensed under the terms of the MIT license.
 
-Feature: Register a system to be managed via SSH push
+Feature: Register a system(traditional and salt) to be managed via SSH push
```

Comment:
i thought that the feature was ok in the same traditional "ssh-push" feature, and not creating a new feature.